docs/developer-guide/local_testing_guide.md

```
@@ -80,17 +79,16 @@ export OVN_SECOND_BRIDGE=[true|false]
 
 You can refer to a recent CI run from any pull request in [https://github.com/ovn-kubernetes/ovn-kubernetes/actions](https://github.com/ovn-kubernetes/ovn-kubernetes/actions) to get a valid set of settings.
 
-As an example for the `control-plane-noHA-local-ipv4-snatGW-1br` job, the settings are at time of this writing:
+As an example for the `control-plane-local-ipv4-snatGW-1br` job, the settings are at time of this writing:
 ```
```

markdownlint-cli2 (0.23.2)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)